src/searchbar.vue

```diff
@@ -0,0 +1,52 @@
+<script>
+    import Typeahead from './Typeahead.vue'
+    export default {
+      extends: Typeahead,
+      computed: {
+        primitiveData() {
+          if (this.data) {
+            var matches = []
+            var words
+            this.data.map(value => {
+              var query = this.value.toLowerCase()
+              var hasHeadingMatch = false
+              var title = value.title.toLowerCase()
+              var keywords = value.keywords? value.keywords.toLowerCase(): ""
```

Comment:
Need to have spacing before the `?` and `:`. I.e.

`value.keywords ? value.keywords.toLowerCase() : ""`

Reply:
Also we use single quotes for strings, `''` rather than `""`.

Reply:
Since this is a new file, let's use eslint.
Let me see if I can configure it. For now, you can copy-paste it into a **.js** file in MarkBind.
We can extend this to all custom files (components) later.